src/content/1.7/development/components/form/types-reference/translatable.md

```diff
@@ -14,6 +14,12 @@ The `TranslatableType` allows you to configure multilanguage input. This multila
 | options | array  | Empty array                                           | Options for configured `type`            |
 | locales | array  | Enabled shop locales (languages)                      | Locales in which field can be translated |
 
+{{% notice info %}}
+If you wish to use [FormattedTextareaType]({{< ref "1.7/development/components/form/types-reference/formatted-textarea/" >}}) as type, your base type must be `TranslateType` instead of `TranslatableType`.
```

Comment:
```suggestion
If you wish to use [FormattedTextareaType]({{< ref "1.7/development/components/form/types-reference/formatted-textarea.md" >}}) as type, your base type must be `TranslateType` instead of `TranslatableType`.
```